package.json

```diff
@@ -30,13 +30,13 @@
     "@babel/plugin-transform-strict-mode": "^7.12.1",
     "@babel/preset-env": "^7.12.1",
     "babel-plugin-add-module-exports": "^1.0.4",
-    "browserslist-config-nebenan": "^1.0.0",
+    "browserslist-config-nebenan": "^1.0.1",
     "chai": "^4.2.0",
     "chalk": "^4.1.0",
     "date-fns": "^2.16.1",
-    "eslint": "^7.13.0",
-    "eslint-config-nebenan-base": "^1.6.0",
-    "lodash": "^4.17.20",
+    "eslint": "^8.31.0",
+    "eslint-config-nebenan-base": "^2.0.2",
+    "lodash": "^4.17.21",
```

Comment:
couldn't we switch to lodash-es here? we already use it in some places and it should be treeshakeable due to being esmodules

Reply:
I would like to try out "normal" lodash first since we use it everywhere else too. Switching only here to lodash-es would IMO not help unless we change it in all other packages too.

Reply:
my thought was that the problem lies in the reqire of the whole lodash. with my PR it should only import like this

var _upperFirst = _interopRequireDefault(require("lodash/upperFirst"));